bitnami/kube-state-metrics/values-production.yaml

```diff
@@ -99,6 +99,7 @@ collectors:
   statefulsets: true
   storageclasses: true
   verticalpodautoscalers: false
+  validatingwebhookconfigurations: true
```

Comment:
I would set this to `false` by default, so we avoid modifications when upgrading to the new version.
The same applies to the `values.yaml` file.

Reply:
I set it to `true` because this is enabled by default upstream. Would you like me to change that? It follows the same convention like the rest of the options where those which are enabled by default upstream are also set to `true` in the chart.

Reply:
Yes, we would like to keep changes (when upgrading) at minimum, and let the user choose when to change the new options.